lib/ansible/modules/group.py

```diff
@@ -605,6 +635,7 @@ def main():
         required_if=[
             ['non_unique', True, ['gid']],
         ],
+        mutually_exclusive=[('local', 'force')]
```

Comment:
I was thinking required_if might work (if local is True then require force is False) but it doesn't seem like this is actually possible with an AnsibleModule parameter. `mutually_exclusive` would make this invalid, which isn't right:
```yaml
group:
  force: false
  local: false
```

It should error if they're both True since the command would be constructed with an invalid option. It could be handled below, along the lines of:
```python
if module.params['force'] and module.params['local']):
    module.fail_json(msg='force is not a valid option for local, force=True and local=True are mutually exclusive')
```